src/Layers/xrRender_R2/r2.cpp

```diff
@@ -220,7 +222,7 @@ void CRender::create()
     Device.seqFrame.Add(this, REG_PRIORITY_HIGH + 0x12345678);
 
     m_skinning = -1;
-#ifndef USE_DX9
+#if defined(USE_DX11) || defined(USE_OGL)
```

Comment:
Better to leave it as is for now.
```suggestion
#ifndef USE_DX9
```